src/models/group.js

```diff
@@ -1,5 +1,30 @@
 import stampit from 'stampit';
 import {Meta, Model} from './base';
+import _ from 'lodash';
+import QuerySet from '../querySet';
+
+const GroupQuerySet = stampit().compose(QuerySet).methods({
+  /**
+  * Fetches Users belonging to a group.
+  * @memberOf GroupQuerySet
+  * @instance
+
+  * @param {Object} properties lookup properties used for path resolving
+  * @returns {GroupQuerySet}
+
+  * @example {@lang javascript}
+  * Grop.please().users({ id: 1, instanceName: 'test-one'}).then(function(users) {});
+  */
+  users(properties = {}) {
+    this.properties = _.assign({}, this.properties, properties);
+
+    this.method = 'GET';
+    this.endpoint = 'users';
+
+    return this;
```

Comment:
maybe we should go with:
```
....
const {User} = this.getConfig();
this.raw = true;
....
return this.then((response) => User.please().serialize(response))
```

this is just a raw idea :P which needs some tuning